lbr_demos/lbr_demos_py/lbr_demos_py/asbr.py

```diff
@@ -0,0 +1,527 @@
+'''Collection of functions/types from the ASBR class taken in Spring 2022
```

Comment:
hi @OmidRezayof and thank you for the PR. My issue here is still that these utilities should not live inside this repository but rather by a Python library that is provided externally.

Reply:
Thank you for your reply @mhubii .
I understand your point. I'm sure there are alternatives for this in external libraries but I couldn't find a suitable one. I can contact the original author of this library and see if he decides to make it as an installable package.
I can change your other comments but for this one I don't think I would have a quick solution. 
Let me know if there was anything I could do! Just wanted to contribute to this repo and help other people who were looking for this functionality! : )